app/controllers/api/v1/config_templates_controller.rb

```diff
@@ -95,7 +95,7 @@ def resource_class
       end
 
       def deprecated
-        ::ActiveSupport::Deprecation.warn('Config templates were renamed to provisioning templates')
+        Foreman::Deprecation.deprecation_warning("1.11", "Config templates were renamed to provisioning templates")
```

Comment:
I don't think as long as the v1 API exists that we should add any hard failures (deadline) for individual routes.. either keep it all, or remove it all.  Nobody's offered up any proposal about when to remove older APIs.